### ISSUE TITLE
The authority of my token is defined on an address that I do not know

### ISSUE THREAD

**Issue by SaplingNetwork** [metaplex-foundation/metaplex#2230]
Hello,

I created 100M tokens for my project last year , i added logo and name with token-list. And this month token-list is deactivated from solana and started to use metadata.

I minted from "Dj5hgRLjFm6WigycUyQpQi7wRgbXV2gWg4Zutaf29gUL"

Today i checked my token on solana explorer and i saw this warning: (And my logo and name removed.)

Warning! Token names and logos are not unique. This token may have spoofed its name and logo to look like another token. Verify the token's mint address to ensure it is correct.

I tried to set metadata with this video but i failed. (https://www.youtube.com/watch?v=ZLtoKzGAjds&t=63s)

I searched errors and i learnt my authority adress is a foreign adress.

**"AqH29mZfQFgRpfwaPoTMWSKJ5kqauoc1FwVBRksZyQrt"**

If you look solana explorer metadata page, you can see it on 3. line. IMPORTANT:this adress has lots of tokens, please [click and check](https://explorer.solana.com/address/AqH29mZfQFgRpfwaPoTMWSKJ5kqauoc1FwVBRksZyQrt)

"updateAuthority":string"AqH29mZfQFgRpfwaPoTMWSKJ5kqauoc1FwVBRksZyQrt"

https://explorer.solana.com/address/E52bRrLGu1YFHBLNTWhdeGoYKyp1UYCTjB7XPoFgapYS

I have only keypair file, i minted tokens but i am not authoritor as i understand it.

What is this address "AqH29mZfQFgRpfwaPoTMWSKJ5kqauoc1FwVBRksZyQrt" can i change this?

**Comment by injaan** [comment]
Same here. who is "AqH29mZfQFgRpfwaPoTMWSKJ5kqauoc1FwVBRksZyQrt" how to update our token metadata

**Comment by SPLTOKEN** [comment]
I have the same issue ,can some one help ?

**Comment by scara89** [comment]
Yes.   “AqH29mZfQFgRpfwaPoTMWSKJ5kqauoc1FwVBRksZyQrt” was @metaplex

https://github.com/metaplex-foundation/metaplex/issues/1055#issuecomment-1262078173

**Comment by injaan** [comment]
@scara89  @SaplingNetwork  Is there any news guys? how to get my token metadata update authority?

**Comment by pranaybathini** [comment]
I am not sure about authority but the metadata including the name and uri can be updated using [metaboss](https://metaboss.rs/create.html )
